stride/assetlist.json

```diff
@@ -366,6 +366,43 @@
         }
       ]
     },
+    {
+      "description": "Stride's liquid staked DYM",
+      "denom_units": [
+        {
+          "denom": "stadym",
+          "exponent": 0
+        },
+        {
+          "denom": "stTIA",
+          "exponent": 18
+        }
+      ],
+      "base": "stadym",
+      "name": "Stride Staked DYM",
+      "display": "stDYM",
+      "symbol": "stDYM",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "dymension_1100-1",
```

Comment:
although this is the chain_id, we need chain_name which is 'dymension'

Reply:
Makes sense, updated!